.gitignore

```diff
@@ -6,3 +6,4 @@
 .vagrant
 !.vagrant/utilities
 .vagrant-user.yaml
+.idea
```

Comment:
Do we really want to add IDE files here?

Reply:
@grzegorzstyla I don't see any downside in doing that

Reply:
ofc otherwise I need to check which files I need to add which is a pain in the ass...